src/app.js

```diff
@@ -17,5 +17,13 @@ app.use(logger("dev"));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
+// For developement purpose
+app.use(function(req, res, next) {
```

Comment:
Well, the `cors` lib is already in our dependency tree, so you can use something like this:
```suggestion
const cors = require("cors");

app.use(cors());  // Enables all CORS requests
```